src/calendar-heatmap.js

```diff
@@ -102,7 +103,17 @@ function calendarHeatmap() {
           var result = cellDate.week() - firstDate.week() + (firstDate.weeksInYear() * (cellDate.weekYear() - firstDate.weekYear()));
           return result * (SQUARE_LENGTH + SQUARE_PADDING);
         })
-        .attr('y', function (d, i) { return MONTH_LABEL_PADDING + d.getDay() * (SQUARE_LENGTH + SQUARE_PADDING); });
+        .attr('y', function (d, i) {
+          var weekDay = d.getDay();
+          if (weekStart === 1) {
```

Comment:
Could you put the logic contained in lines 108 - 114 into a separate function so we can re-use it and keep the code DRY?